src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -14,38 +16,37 @@ import addEncryptedAuthTokenToURL from '@libs/addEncryptedAuthTokenToURL';
 import * as Browser from '@libs/Browser';
 import * as DeviceCapabilities from '@libs/DeviceCapabilities';
 import CONST from '@src/CONST';
-import {videoPlayerDefaultProps, videoPlayerPropTypes} from './propTypes';
 import shouldReplayVideo from './shouldReplayVideo';
+import type {VideoPlayerProps, VideoWithOnFullScreenUpdate} from './types';
 import * as VideoUtils from './utils';
 import VideoPlayerControls from './VideoPlayerControls';
 
 const isMobileSafari = Browser.isMobileSafari();
 
 function BaseVideoPlayer({
     url,
-    resizeMode,
-    onVideoLoaded,
-    isLooping,
+    resizeMode = ResizeMode.CONTAIN,
+    onVideoLoaded = () => {},
+    isLooping = false,
     style,
     videoPlayerStyle,
     videoStyle,
     videoControlsStyle,
     videoDuration,
-    shouldUseSharedVideoElement,
-    shouldUseSmallVideoControls,
-    shouldShowVideoControls,
-    onPlaybackStatusUpdate,
-    onFullscreenUpdate,
+    shouldUseSharedVideoElement = false,
+    shouldUseSmallVideoControls = false,
+    shouldShowVideoControls = true,
+    onPlaybackStatusUpdate = () => {},
+    onFullscreenUpdate = () => {},
```

Comment:
I think the default values for a callbacks are not needed

Reply:
Are you sure? Without them we might invoke object which is possibly undefined. Those props are optional as some BaseVideoPlayer calls does not provide them.

Reply:
yeah, and when calling those functions you can use optional chaining eg. `onPlaybackStatusUpdate.?()` then if the function is undefined nothing will happen 😄

Reply:
Oh yes, that's right. I have removed default callbacks then.